packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.queue-processing-fargate-service.ts

```diff
@@ -141,6 +141,7 @@ export = {
       image: ecs.ContainerImage.fromRegistry('test'),
       maxReceiveCount: 42,
       retentionPeriod: cdk.Duration.days(7),
+      visibilityTimeout: cdk.Duration.minutes(5),
```

Comment:
looks great -- can you add a similar test modification for the Ec2 version of this? https://github.com/aws/aws-cdk/blob/85ba0361febc44edb66577afa8ba55ad466e32d6/packages/%40aws-cdk/aws-ecs-patterns/test/ec2/test.queue-processing-ecs-service.ts#L109